lib/api/src/exports.rs

```diff
@@ -3,6 +3,7 @@ use crate::import_object::LikeNamespace;
 use crate::native::NativeFunc;
 use crate::WasmTypeList;
 use indexmap::IndexMap;
+use loupe_derive::MemoryUsage;
```

Comment:
You can reexport this through `loupe` so users only need to depend on one crate and things work much more smoothly.

In fact, I'd recommend making it a default feature of `loupe` to reexport the derive macro and you can opt-out by disabling default features.

Reply:
That's a good idea!